langstream-runtime/langstream-runtime-impl/src/main/python/Pipfile

```diff
@@ -6,7 +6,7 @@ fastavro = "*"
 "openai[embeddings]" = "*"
 "openai[datalib]" = "*"
 "openai[wandb]" = "*"
-langchain = "*"
+"langchain[openai]" = "*"
```

Comment:
This change seems unrelated

Reply:
Done instead in https://github.com/LangStream/langstream/pull/723